dev-cpp/gtest-parallel/gtest-parallel-20220621.ebuild

```diff
@@ -0,0 +1,34 @@
+# Copyright 2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{13..14} )
+inherit python-r1
+
+DESCRIPTION="Run Google Test suites in parallel."
+HOMEPAGE="https://github.com/google/gtest-parallel"
+COMMIT="f4d65b555894b301699c7c3c52906f72ea052e83"
```

Comment:
Please use MY_COMMIT, or something else less generic. Many ebuilds use that but the generic name has caused issues in the past and would rather avoid it for new ebuilds.